kombu/transport/redis.py

```diff
@@ -124,7 +124,8 @@ def Mutex(client, name, expire):
             try:
                 with client.pipeline(True) as pipe:
                     pipe.watch(name)
-                    if pipe.get(name) == lock_id:
+                    pipename = pipe.get(name)
+                    if pipename and pipename.decode() == lock_id:
```

Comment:
Kombu has already some utility functions for these common use cases. I think here we could use [bytes_to_str](https://github.com/celery/kombu/blob/fe5adb539ce9b351ffe3df0719ca281a736b0014/kombu/utils/encoding.py#L53).

Reply:
Good idea